toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/theme/ScalebarTheme.kt

```diff
@@ -114,19 +113,20 @@ public object ScalebarDefaults {
 
     /**
      * Creates an instance of [ScalebarShapes] with default values.
-     *
-     * @param shadowCornerRadius The corner radius of the shadow on the Scalebar.
+     * [textShadowBlurRadius] is set to 0.0, meaning no shadow blur will be applied. [barCornerRadius] is set to 0.0,
+     * resulting in a square corner.
+     * @param textShadowBlurRadius The corner radius of the shadow on the Scalebar.
```

Comment:
```suggestion
     * @param textShadowBlurRadius The blur radius on the shadow of all text.
```
also can you add a newline above this